src/fetchers.js

```diff
@@ -110,25 +85,27 @@ function getDefault(url) {
  * @param {string[]} indices - array of indices.
  * @returns {PathInfo[]} An array of path info structures.
  */
-function getPathInfos(urlPath, mount, indices, resolveDefault = ((a) => a)) {
+function getPathInfos(urlPath, mount, indices) {
   // eslint-disable-next-line no-param-reassign
   urlPath = urlPath.replace(/\/+/, '/');
   // check if url has extension, and if not create array of directory indices.
   const urls = [];
+  const extensionlessurls = [];
```

Comment:
AFAICS, this array is never used ?

Reply:
Indeed, this line https://github.com/adobe/helix-dispatch/pull/247/files#diff-fb1ce3759626b7da699b6918ce56f3a6R137 was merged away.